packages/mobile/src/firebase/firebase.ts

```diff
@@ -284,6 +284,8 @@ export async function fetchRemoteFeatureFlags(): Promise<RemoteFeatureFlags | nu
       multiTokenShowHomeBalances: flags.multiTokenShowHomeBalances.asBoolean(),
       multiTokenUseSendFlow: flags.multiTokenUseSendFlow.asBoolean(),
       multiTokenUseUpdatedFeed: flags.multiTokenUseUpdatedFeed.asBoolean(),
+      paymentDeepLinkHandler: '',
```

Comment:
I can help you set up a feature flag so we can use the real code, reach out to me on Slack and we'll look into it :)